### ISSUE TITLE
Uncaught (in promise): [object Null] only if I'm not debugging

### ISSUE THREAD

**Issue by silentsnooc** [Enngage/ngx-captcha#61]
I use

```json
"ngx-captcha": "^7.0.0",
``` 
and all I do is:

```typescript
    this._reCaptchaV3Service.execute(
      environment.reCaptchaV3SiteKey, 'register',
      (reCaptchaToken) => {
        console.log(reCaptchaToken);
        this.form.patchValue({reCaptchaToken: reCaptchaToken});
        this.loadingReCaptchaToken = false;
      });
```

Now, if I debug, everything works fine. 

But if I just run the app, what I get is:

```
Error: Uncaught (in promise): [object Null]
    at resolvePromise (zone.js:814) [<root>]
    at resolvePromise (zone.js:771) [<root>]
    at :4201/polyfills.js:3218:17 [<root>]
    at drainMicroTaskQueue (zone.js:595) [<root>]
    at ZoneTask.push../node_modules/zone.js/dist/zone.js.ZoneTask.invokeTask [as invoke] (zone.js:500) [<root>]
    at invokeTask (zone.js:1540) [<root>]
    at MessagePort.globalZoneAwareCallback (zone.js:1566) [<root>]
api.onUnhandledError @ zone.js:668
handleUnhandledRejection @ zone.js:692
_loop_1 @ zone.js:682
api.microtaskDrainDone @ zone.js:686
drainMicroTaskQueue @ zone.js:602
push../node_modules/zone.js/dist/zone.js.ZoneTask.invokeTask @ zone.js:500
invokeTask @ zone.js:1540
globalZoneAwareCallback @ zone.js:1566
```

Any idea how to fix this?

**Comment by Enngage** [comment]
Hard to say, the stacktrace you posted doesn't contain any of my code.. I wouldn't rule out issue with something else entirely.

**Comment by silentsnooc** [comment]
@Enngage Well, it was my fault. I added `localhost` to the allowed domains for my dev key. However, as I got redirected somewhere it changed localhost to my local IP and well.

Since I added this CSS

```
// Hide Google reCaptcha badge
// Do not use "display: None" as it disabled spam checking (I've been told)
.grecaptcha-badge { visibility: hidden; }
```

I wasn't able to see the issue ..

Idk if it would be possible to detect such a problem and prompt an additional alert or write that to the console.

Anyway, this issue can be closed.
